school-finder-frontend/app/components/nav/NavController.js

```diff
@@ -0,0 +1,13 @@
+(function () {
+  var sfApp = angular.module('sfApp');
+
+  var NavCtrl = function($scope, $location){
+      $scope.isActive = function (viewLocation) {
+          return viewLocation === $location.path().replace(/^(?:\/)+/, "");;
```

Comment:
I believe UI-sref-active could replace this function. http://angular-ui.github.io/ui-router/site/#/api/ui.router.state.directive:ui-sref-active 
Also, Two semicolons.